testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCreateTableAsSelectCompatibility.java

```diff
@@ -172,9 +171,7 @@ public void testReplaceTableWithSchemaChange()
         }
     }
 
-    // Databricks 11.3, 12.2 and 13.3 don't create a checkpoint file at 'CREATE OR REPLACE TABLE' statement
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_133, PROFILE_SPECIFIC_TESTS})
-    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
```

Comment:
So does this means - `Replace table` doesn't create a new checkpoint on Databricks side ?

Reply:
Yes. Or it might be a bug in assertLastEntryIsCheckpointed method.

Reply:
Bug as in ?